src/NuGet.Clients/NuGet.Tools/NuGetTools.vsct

```diff
@@ -231,6 +231,14 @@
     <GuidSymbol name="guidNuGetDebugConsoleCmdSet" value="{DDC61543-6CA7-4A6F-A5B7-984BE723C52F}">
       <IDSymbol name="cmdidDebugConsole" value="0x0900" />
     </GuidSymbol>
+    <GuidSymbol name="PackageIconMonikers" value="{8F5EAE8F-9892-4CE2-826C-764BEDE6D2EC}">
```

Comment:
What's the purpose of adding these here? And I see other values are hexa decimal, so will that make any difference?